library/spark/spark-library/src/test/java/uk/gov/gchq/gaffer/spark/serialisation/kryo/impl/FreqMapKryoSerializerTest.java

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Is there an opportunity here to create a KryoSerializerTest which is extended by the other test classes? The implementation details in each of the test methods are very similar.